content/courses/program-optimization/program-architecture.md

```diff
@@ -820,64 +820,75 @@ engine in Solana. This program will have the following features:
 - Spent action points go to a game's treasury as listed in the `Game` account
 
 We'll walk through the tradeoffs of various design decisions as we go to give
-you a sense for why we do things. Let’s get started!
+you a sense of why we do things. Let's get started!
 
-#### 1. Program Setup
+### 1. Program Setup
 
 We'll build this from scratch. Start by creating a new Anchor project:
 
 ```bash
 anchor init rpg
 ```
 
-<Callout type="note">This lab was created with Anchor version `0.28.0` in mind.
-If there are problems compiling, please refer to the
-[solution code](https://github.com/Unboxed-Software/anchor-rpg/tree/challenge-solution)
-for the environment setup.</Callout>
+<Callout type="note">
+
+This lab was created with Anchor version `0.30.1` in mind. If there are problems
+compiling, please refer to the
+[solution code](https://github.com/Unboxed-Software/anchor-rpg/tree/main) for
+the environment setup.</Callout>
 
 Next, replace the program ID in `programs/rpg/lib.rs` and `Anchor.toml` with the
-program ID shown when you run `anchor keys list`.
+program ID shown when you run `anchor keys list`. Alternatively, you can run
+command `anchor keys sync` that will automatically sync your program ID. This
+command will sync the program ids between the program files(including
```

Comment:
Just use `sync` rather than tell them to edit the files. One simple direction forward. No unnecessary choices.

Reply:
Updated the content to give simple direction.